iss_forecasting/analysis/utils/plotting.py

```diff
@@ -0,0 +1,152 @@
+"""Analysis utils for plotting
+"""
+import altair as alt
+import matplotlib.pyplot as plt
+from matplotlib.offsetbox import AnchoredText
+import seaborn as sns
+import math
+
+# Set Matplotlib defaults
+plt.style.use("seaborn-whitegrid")
+plt.rc("figure", autolayout=True, figsize=(11, 4))
+plt.rc(
+    "axes",
+    labelweight="bold",
+    labelsize="large",
+    titleweight="bold",
+    titlesize=16,
+    titlepad=10,
+)
+plot_params = dict(
+    color="0.75",
+    style=".-",
+    markeredgecolor="0.25",
+    markerfacecolor="0.25",
+)
+
+
+def plot_two_y_one_x(
+    data_source,
+    x,
+    y1,
+    y2,
+    chart_title,
+    line_1_colour="#5276A7",
+    line_2_colour="#57A44C",
+):
```

Comment:
Hey, would it be OK to use type hints in the definitions of function? (as explained [here](https://docs.python.org/3/library/typing.html)). Do let me know if you have objection to this style.

For the LayerChart perhaps even makes sense to use a type alias (eg, if this is going to be the output type for multiple functions)

For example:

```suggestion
def plot_two_y_one_x(
    data_source: pd.DataFrame,
    x: str,
    y1: str,
    y2: str,
    chart_title: str,
    line_1_colour: str="#5276A7",
    line_2_colour: str="#57A44C",
) -> altair.vegalite.v4.api.LayerChart:
```

Reply:
For the `LayerChart` perhaps even makes sense to use a type alias (eg, if this is going to be the output type for multiple functions)

Reply:
For the LayerChart perhaps even makes sense to use a type alias (eg, if this is going to be the output type for multiple functions)

Reply:
Is this so it is more readable? If so, I have added `from altair.vegalite.v4.api import LayerChart` at the top of the file so just `LayerChart` can be used here.